core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -55,15 +54,15 @@ THE SOFTWARE.
                       ${%Idle}
                     </j:otherwise>
                   </j:choose>
-                </td>
-                <td class="pane"/>
-                <td class="pane"/>
+                </div>
               </j:when>
               <j:otherwise>
                 <!-- not actually optional, but it helps with backward compatibility -->
                 <j:set var="executor" value="${e}" />
+                <table>
```

Comment:
This should be avoided, but plugins uses executorCell.jelly starting directly with <td>. I think this could be improved with an alternate "executorCell" (with fluid syntax) or PR to any plugin that have executorCell.jelly file